src/prelaunch_data_generator/prelaunch_data_generator_launcher/prelaunch_data_generator_launcher.star

```diff
@@ -1,4 +1,4 @@
-IMAGE = "ethpandaops/ethereum-genesis-generator:1.3.1"
+IMAGE = "ethpandaops/ethereum-genesis-generator:2.0.0-rc.1"
```

Comment:
Please change it to `IMAGE = "ethpandaops/ethereum-genesis-generator:2.0.0-rc.6"`